composer.json

```diff
@@ -20,12 +20,12 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.2",
```

Comment:
Why would php 8.2 be required? I think we can keep supporting version 8.0 as well. When used in Laravel 12, you will automatically get the correct version installed.